contracts/PFA.sol

```diff
@@ -144,5 +133,6 @@ abstract contract PFA is IPFA, LimitedOwnable {
         _licenseTimestamps[recipient_] = block.timestamp;
         emit License(recipient_);
         _transactionCount++;
+        payable(owner()).transfer(msg.value);
```

Comment:
Can you provide detail on what the explicit cost increase will be here when adding this to the contract? We should assess whether we'll need to increase the cost estimation at playlist upload time when calling the license function. This should be small, but we need to be precise.

Reply:
We dynamically evaluate the cost of the deployment via the abi, so I don't think we would need to make any hardcoded changes as any change in the contract would be reflected in the compiled ABI and then in the total cost.

Reply:
https://github.com/formless-eng/share/blob/bcd850a4d3eced0876b8159b154885fdf09d686a/backend/server.js#L691

Here if you follow the definition of calculateGNFTDeploy cost you will find the dynamic estimation of gas cost.